draft-ietf-keytrans-architecture.md

```diff
@@ -400,6 +400,94 @@ Alice                  Transparency Log                  Manager
 Transparency Log to the Manager. Rejected requests are blocked." }
 
 
+# Combining Logs
+
+There are many cases where it makes sense to operate multiple cooperating log
+instances, for example:
+
+- A service provider may decide that it's prudent to rotate its cryptographic
+  keys, or migrate to a new deployment mode. They can do this by creating a new
+  log instance with new cryptographic keys, operating under a new
+  deployment mode if desired, and migrating their data from the old log to the
+  new log while users are able to query both.
+- A service provider may choose to operate multiple logs to improve their
+  ability to scale or provide higher availability.
+- A federated system may allow each participant in the federation to operate
+  their own log for their own users.
+
+Client implementations should generally be prepared to interact with multiple
+logs simultaneously. In particular, clients SHOULD namespace any configuration
+or state related to a particular log, such that information related to different
+logs do not conflict.
+
+When multiple logs are used, all users in the system MUST have a consistent
+policy for executing Search, Update, and Monitor queries against the logs in a
+way that maintains the high-level security guarantees of KT:
+
+- If all logs behave honestly, then users observe a globally-consistent view of
+  the data associated with each key.
+- If any log behaves dishonestly such that the prior guarantee is not met (some
+  users observe data associated with a key that others do not), this will be
+  detected either immediately or in a timely manner by background monitoring.
+
+## Gradual Migration
+
+In the case of gradually migrating from an old log to a new one, this policy may
+look like:
+
+1. Search queries should be executed against the old log first, and then against
+   the new log only if the most recent version of a key in the old log is a
+   tombstone.
```

Comment:
"Tombstone" is used here for the first time without further definition. I think this is a concept that was first introduced in SEEMless and requires introduction.